### ISSUE TITLE
Error/bug with OF BW

### ISSUE THREAD

**Issue by akouSDN** [RDelg/OF-BW-Control#3]
Hi @RDelg,
The RYU gives this error when number of hosts per-switch increase to 10: 

hub: uncaught exception: Traceback (most recent call last):
  File "/home/mininet/ryu/ryu/lib/hub.py", line 52, in _launch
    func(_args, *_kwargs)
  File "/home/mininet/ryu/ryu/base/app_manager.py", line 287, in _event_loop
    handler(ev)
  File "/home/mininet/ryu/ryu/app/OF-BW-Control/Switch.py", line 137, in _meter_stats_reply_handler
    if (self.rate_used[dpid][port][src] >= int(self.rate_allocated[dpid][port][src]*0.7)
KeyError: 4294967294

It seems like a memory management problem ! or whet do you think! 

Furthermore if we use it to monitor 2 Userswitch, its also gives the same issue

Regards,

**Comment by RDelg** [comment]
Hi @akouSDN,

Your error occur when the handler tries to access to the port 4294967294. Once I saw that number too and I took a screenshot of it:

![4294967294](https://cloud.githubusercontent.com/assets/7086947/12383642/2401ece4-bd87-11e5-99f4-ed607e953c58.png)

Apparently this port is something particular, so searching I found [this](http://comments.gmane.org/gmane.network.ryu.devel/9752). 

Port 4294967294 is a local port for OpenFlow so we just have to ignore it with a
[exception](https://github.com/RDelg/OF-BW-Control/commit/337488643c4220429de6d1de5b294f71de128fc9).

Regards.
